packages/yarnpkg-pnp/sources/loader/makeApi.ts

```diff
@@ -819,7 +819,20 @@ export function makeApi(runtimeState: RuntimeState, opts: MakeApiOptions): PnpAp
       const containingPackage = findPackageLocator(unqualifiedPath);
       if (containingPackage) {
         const {packageLocation} = getPackageInformationSafe(containingPackage);
-        if (!opts.fakeFs.existsSync(packageLocation)) {
+
+        let exists = true;
+        try {
+          opts.fakeFs.accessSync(packageLocation);
+        } catch (err) {
+          if (err.code === `ENOENT`) {
+            exists = false;
+          } else {
+            const readableError: string = (err.message ?? err ?? `empty exception thrown`).replace(/^[A-Z]/, ($0: string) => $0.toLowerCase());
```

Comment:
`err` can't be `null | undefined` here, if it was we would get a `TypeError` when checking `err.message`